src/agent/protocols/hack/update-generated-proto.sh

```diff
@@ -1,68 +1,133 @@
 #!/bin/bash
 
 die() {
+    echo "===================================================================="
```

Comment:
Nit: since this function is an error function, the output should really go to `stderr`.